### DISCUSSION TITLE
Stop assuming `bash` exists in scripts

### DISCUSSION THREAD

**Discussion by aleclarson** [radashi-org/radashi#311] [Ideas]
Not everyone has Bash on their dev machine, so let's replace the Bash scripts in [./scripts/](https://github.com/radashi-org/radashi/tree/main/scripts) with Node.js. This will ease contribution without having to install a tool like Bash beforehand.

**Comment by MarlonPassos-git** [comment]
I think you've already[ solved this yourself](https://github.com/radashi-org/radashi/pull/315) 😄, good job
